app/models/katello/concerns/smart_proxy_extensions.rb

```diff
@@ -62,14 +62,22 @@ def refresh
         }
         scope :with_content, -> { with_features(PULP_FEATURE, PULP_NODE_FEATURE) }
 
-        def self.default_capsule
+        def self.pulp_master
           unscoped.with_features(PULP_FEATURE).first
         end
 
+        def self.pulp_master!
+          proxy = pulp_master
+          fail _("Could not find a smart proxy with pulp feature.") if proxy.nil?
+          proxy
+        end
+
+        def self.default_capsule
+          pulp_master
+        end
+
         def self.default_capsule!
-          capsule = default_capsule
-          fail _("Could not find a smart proxy with pulp feature.") if capsule.nil?
-          capsule
+          pulp_master!
```

Comment:
Again, I'd prefer an explicit alias here

Reply:
same issue with class method aliases

Reply:
there are ways to do it from googling, but none are 'built in' and more hacky than this IMO (unless i'm missing some new trick)

Reply:
Yeah, forget about aliasing class methods. I didn't really realize that. TIL! :)